python/ccf/proposal_generator.py

```diff
@@ -299,6 +301,30 @@ def update_recovery_shares(**kwargs):
 def set_recovery_threshold(threshold, **kwargs):
     return build_proposal("set_recovery_threshold", threshold, **kwargs)
 
+@cli_proposal
+def update_root_ca_cert(cert_name, cert_path, skip_checks=False, **kwargs):
+    cert_pem = open(cert_path).read()
```

Comment:
I prefer to see fds scoped explicitly.

Reply:
Alright, just copied from other functions in the same file.